routes/v1.js

```diff
@@ -7,38 +7,85 @@ var base        = require('taskcluster-base');
 // Common schema prefix
 var SCHEMA_PREFIX_CONST = 'http://schemas.taskcluster.net/aws-provisioner/v1/';
 
+
 /**
  * API end-point for version v1/
  *
  * In this API implementation we shall assume the following context:
  * {
  *   publisher:         // Publisher created with exchanges.js
- *   workerType:        // Instance of data.WorkerType
+ *   WorkerType:        // Instance of data.WorkerType
  * }
  */
 var api = new base.API({
   title:          "AWS Provisioner API Documentation",
   description: [
-    "The AWS  provisioner, typically available at",
-    "`aws-provisioner.taskcluster.net`, is responsible for provisioning EC2",
-    "instances as tasks become become pending. To do this it monitors the",
-    "state of queues, EC2 instances, spot prices and other interesting",
-    "parameters."
-    // TODO: Write who the intended users are:
-    //        - tools for defining workerTypes
-    //        - tools for inspecting state of AWS
-    //        - hooks monitoring state
-    // TODO: Write what the AWS provisioner does, how the API works
-    //       just some introduction...
-    //       To quote a the Daleks: Explain..., Explain,.. Exterminate!
-    //                                                    (^disregard the that)
+    "The AWS Provisioner is responsible for provisioning instances on EC2 for use in",
+    "TaskCluster.  The provisioner maintains a set of worker configurations which",
+    "can be managed with an API that is typically available at",
+    "aws-provisioner.taskcluster.net.  This API can also perform basic instance",
+    "management tasks in addition to maintaining the internal state of worker type",
+    "configuration information.",
+    "",
+    "The Provisioner runs at a configurable interval.  Each iteration of the",
+    "provisioner fetches a current copy the state that the AWS EC2 api reports.  In",
+    "each iteration, we ask the Queue how many tasks are pending for that worker",
+    "type.  Based on the number of tasks pending and the scaling ratio, we may",
+    "submit requests for new instances.  We use pricing information, capacity and",
+    "utility factor information to decide which instance type in which region would",
+    "be the optimal configuration.",
+    "",
+    "Each EC2 instance type will declare a capacity and utility factor.  Capacity is",
+    "the number of tasks that a given machine is capable of running concurrently.",
+    "Utility factor is a relative measure of performance between two instance types.",
+    "We multiply the utility factor by the spot price to compare instance types and",
+    "regions when making the bidding choices.",
+    "",
```

Comment:
I don't really care about 80 vs. 120... but I strongly prefer consistency and we have 80 chars everywhere else.
It's also the general style guideline as I see it:
https://developer.mozilla.org/en-US/docs/Mozilla/Developer_guide/Coding_Style#Line_Length
I see no need to rush to fix it... but going forward I think 80 is the sane standard we do everywhere.
If you want to define a style guide I'm open to that, but I like being as close to the mozilla coding style guideline as possible; granted it's not very specific about javascript.

I just noticed that we recommend javadoc tags... So I guess that settles my documentation issues. If I can find something that can document ES6/7 with that kind of tags. Otherwise I'll probably keep using the obscure/undefined strategy we have now :)